layout-server/src/main/java/org/finos/vuu/layoutserver/service/LayoutService.java

```diff
@@ -0,0 +1,47 @@
+package org.finos.vuu.layoutserver.service;
+
+import lombok.RequiredArgsConstructor;
+import org.finos.vuu.layoutserver.model.Layout;
+import org.finos.vuu.layoutserver.model.Metadata;
+import org.finos.vuu.layoutserver.repository.LayoutRepository;
+import org.finos.vuu.layoutserver.repository.MetadataRepository;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+@RequiredArgsConstructor
+@Service
+public class LayoutService {
+
+    private final LayoutRepository layoutRepository;
+    private final MetadataRepository metadataRepository;
+
+    public Layout getLayout(UUID id) {
+        return layoutRepository.findById(id).orElseThrow();
+    }
+
+    public List<Metadata> getMetadata() {
+        List<Metadata> metadata = new ArrayList<>();
+        layoutRepository.findAll().forEach(layout -> metadata.add(layout.getMetadata()));
```

Comment:
Any reason you're not using `metadataRepository` here?

Reply:
That's on my TODO list, I wrote this before `metadataRepository` existed 😄 Lets leave this comment open until I've changed it so I don't forget.

Reply:
It now uses `metadataRepository`